src/utils.ts

```diff
@@ -0,0 +1,13 @@
+import {EntityPointer, EntityType} from './types';
+
+export const matchesEntityPointer = (entityPointer: EntityPointer, entityType: EntityType, entityId: string): boolean => {
+    return entityPointer.entityType === entityType && entityPointer.entityId === entityId;
+}
```

Comment:
This function is only for making code that calls this shorter. Not sure how much value it brings

Reply:
I'm more concerned with how readable something is, both in terms of brevity and in terms of how clear it is what the code is doing, without any comments. Essentially using variable names such that the code itself is almost like reading comments.

What do you think?

Reply:
Yeah I'm in full agreement. So how do you feel about that in the context of this code we're commenting on?